include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -491,16 +491,19 @@ namespace LinearAlgebra
 
       /**
        * Set each block of this vector as follows:
-       * $U^i = \sum_{j} V_j A^{ji}$ where $U^i$
+       * $U^i = \sum_{j} V_j A^{ji} b$ where $U^i$
        * and $V_j$ indicate the $i$th block (not element) of $U$ and the
-       * $j$th block of $V$, respectively.
+       * $j$th block of $V$, respectively. If @p adding is <code>true</code>,
+       * the result will be added instead.
        *
        * Obviously, this function can only be used if all blocks of both vectors
        * are of the same size.
        */
       template <typename FullMatrixType>
       void mmult(const BlockVector<Number> &V,
-                 const FullMatrixType &matrix);
+                 const FullMatrixType &matrix,
+                 const bool adding = false,
```

Comment:
this optional parameter is first as I expect it to be changed more often than the second one.